frontend/src/scenes/data-warehouse/new/sourceWizardLogic.tsx

```diff
@@ -1373,7 +1373,8 @@ export const getErrorsForFields = (
         errorsObj: Record<string, any>
     ): void => {
         if (field.type === 'switch-group') {
-            if (valueObj[field.name]?.['enabled']) {
+            // handle string value coming down from the backend for an update
+            if (valueObj[field.name]?.['enabled'] && valueObj[field.name]?.['enabled'] !== 'False') {
```

Comment:
Can we check both the string and the boolean? Have you tested if this still works with new sources?